### PR TITLE
Add retries to URI

### LINKED ISSUE #458: "status": -1, "url": "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.4/release.txt"
```
TASK [node-prep : Get Release.txt File] ****************************************
Wednesday 17 June 2020  08:30:22 +0000 (0:00:00.270)       0:00:11.439 ******** 
fatal: [kni1-bootstrap.example.com -> kni1-bootstrap.example.com]: FAILED! => {"changed": false, "content": "", "elapsed": 30, "failed_when_result": true, "msg": "Status code was -1 and not [200]: Request failed: <urlopen error _ssl.c:880: The handshake operation timed out>", "redirected": false, "status": -1, "url": "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.4/release.txt"}
```

This is just a flake but I guess it would be nice to add retries to URLs to avoid those flakes.

### PR DESCRIPTION
**Pull request by iranzo**
# Description

Add retries to URI calls

Closes #458

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by iranzo** [comment]
[test-ipi]

File: ansible-ipi-install/roles/node-prep/tasks/10_validation.yml [hunk 1]
```diff
@@ -157,6 +157,9 @@
     status_code: [-1,200]
     timeout: 1
   register: check_url
+  until: _result.status == 200
```
Comment (rlopez133) [inline ansible-ipi-install/roles/node-prep/tasks/10_validation.yml:160]: not sure but shouldn't this be check_url.status?

Comment (iranzo) [reply ansible-ipi-install/roles/node-prep/tasks/10_validation.yml]: Should be changed in all items :) (thanks!)

**Comment by iranzo** [comment]
[test-ipi]

**Pull request merged**